pytorch_lightning/utilities/distributed.py

```diff
@@ -289,7 +289,7 @@ def register_ddp_comm_hook(
         if not _TORCH_GREATER_EQUAL_1_9:
             rank_zero_warn("Not applying DDP comm wrapper. To use communication wrapper, please use pytorch>=1.9.0.")
         else:
-            rank_zero_info(
+            new_rank_zero_info(
```

Comment:
@ananthsub Why calling it new_rank_zero_info ?

Reply:
to avoid conflicts with the `rank_zero_info` defined elsewhere in this module, which was provided for backwards compatibility.

if i call `rank_zero_info`, i will call `utilities.distributed.rank_zero_info` which triggers a deprecation warning telling me to switch to using `utilities.rank_zero.rank_zero_info`